### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Jamulus/Startpage](https://hosted.weblate.org/projects/jamulus/1-index/).

It also includes the following components:

* [Jamulus/Copyright notices](https://hosted.weblate.org/projects/jamulus/copyright-notices/)

* [Jamulus/Software-Manual](https://hosted.weblate.org/projects/jamulus/software-manual/)

* [Jamulus/Include-Shared-Commands](https://hosted.weblate.org/projects/jamulus/include-shared-commands/)

* [Jamulus/Include-Backing-Up](https://hosted.weblate.org/projects/jamulus/include-backing-up/)

* [Jamulus/General strings](https://hosted.weblate.org/projects/jamulus/general/)

* [Jamulus/Unregistered-Servers](https://hosted.weblate.org/projects/jamulus/unregistered-servers/)

* [Jamulus/Command-Line-Options](https://hosted.weblate.org/projects/jamulus/command-line-options/)

* [Jamulus/Include-Client-Commands](https://hosted.weblate.org/projects/jamulus/include-client-commands/)

* [Jamulus/Server-Troubleshooting](https://hosted.weblate.org/projects/jamulus/server-troubleshooting/)

* [Jamulus/QOS-Windows](https://hosted.weblate.org/projects/jamulus/qos-windows/)

* [Jamulus/Server-Bandwidth](https://hosted.weblate.org/projects/jamulus/server-bandwidth/)

* [Jamulus/Contribution](https://hosted.weblate.org/projects/jamulus/contribution/)

* [Jamulus/Installation-for-Macintosh](https://hosted.weblate.org/projects/jamulus/installation-for-macintosh/)

* [Jamulus/Getting-Started](https://hosted.weblate.org/projects/jamulus/getting-started/)

* [Jamulus/Installation-for-Android](https://hosted.weblate.org/projects/jamulus/installation-for-android/)

* [Jamulus/Installation-for-iOS](https://hosted.weblate.org/projects/jamulus/installation-for-ios/)

* [Jamulus/Client-Troubleshooting](https://hosted.weblate.org/projects/jamulus/client-troubleshooting/)

* [Jamulus/FAQ](https://hosted.weblate.org/projects/jamulus/faq/)

* [Jamulus/Tips-Tricks-More](https://hosted.weblate.org/projects/jamulus/tips-tricks-more/)

* [Jamulus/Navigation](https://hosted.weblate.org/projects/jamulus/navigation/)

* [Jamulus/Installation-for-Linux](https://hosted.weblate.org/projects/jamulus/installation-for-linux/)

* [Jamulus/Installation-for-Windows](https://hosted.weblate.org/projects/jamulus/installation-for-windows/)

* [Jamulus/Running-a-Server](https://hosted.weblate.org/projects/jamulus/running-a-server/)

* [Jamulus/Directories](https://hosted.weblate.org/projects/jamulus/directories/)

* [Jamulus/Privacy-Statement](https://hosted.weblate.org/projects/jamulus/privacy-statement/)

Current translation status:

Weblate translation status

### CODE REVIEW

File: _translator-files/po/fr/Running-a-Server.po [hunk 1]
```diff
@@ -386,10 +386,8 @@ msgstr "`-e ou --directoryserver`"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:140
-#, fuzzy
-#| msgid "Required for a [Registered Server](Running-a-Server#2-registered). Also required to run Jamulus as a [Directory](Directories)."
 msgid "Required for a [Registered Server](Running-a-Server#registered-mode). Also required to run Jamulus as a [Directory](Directories)."
-msgstr "Nécessaire pour un [serveur enregistré](Running-a-Server#2-registered). Également requis pour exécuter Jamulus en tant qu'[annuaire](Directories)."
+msgstr "Nécessaire pour un [serveur enregistré](Running-a-Server#1-enregistré). Également requis pour exécuter Jamulus en tant qu'[annuaire](Directories)."
```
Comment (ignotus666) [inline _translator-files/po/fr/Running-a-Server.po]: This link is wrong

File: _translator-files/po/fr/Directories.po [hunk 2]
```diff
@@ -53,8 +53,6 @@ msgstr "Administrer un annuaire"
 
 #. type: Plain text
 #: ../wiki/en/Directories.md:14
-#, fuzzy
-#| msgid "This is a specialised Jamulus Server configuration, as described in [Server Types](Running-a-Server#server-modes)."
 msgid "This is a specialised Jamulus Server configuration, as described in [Server Modes](Running-a-Server#server-modes)."
 msgstr "Il s'agit d'une configuration spécialisée du serveur Jamulus, comme décrit dans [Types de serveurs](Running-a-Server#types-de-serveur)."
```
Comment (ignotus666) [inline _translator-files/po/fr/Directories.po]: This link is wrong.

File: _translator-files/po/fr/Running-a-Server.po (hunk 1 above)
Comment (trebmuh) [reply _translator-files/po/fr/Running-a-Server.po]: Unless I miss something, the link is good:
https://jamulus.io/fr/wiki/Running-a-Server#1-enregistré

File: _translator-files/po/fr/Directories.po (hunk 2 above)
Comment (trebmuh) [reply _translator-files/po/fr/Directories.po]: Same here : https://jamulus.io/fr/wiki/Running-a-Server#types-de-serveur

File: _translator-files/po/fr/Running-a-Server.po (hunk 1 above)
Comment (ignotus666) [reply _translator-files/po/fr/Running-a-Server.po]: It's correct for the current, _active_ version of the website, but not for the next-release version, which is what we're translating now - see the link in English is `[Registered Server](Running-a-Server#registered-mode)` and not `[Registered Server](Running-a-Server#2-registered)` (what it used to be).

File: _translator-files/po/fr/Directories.po (hunk 2 above)
Comment (ignotus666) [reply _translator-files/po/fr/Directories.po]: As above. What used to be "Server types" is now "Server modes".

File: _translator-files/po/fr/Running-a-Server.po (hunk 1 above)
Comment (trebmuh) [reply _translator-files/po/fr/Running-a-Server.po]: ok, fixed I think.

File: _translator-files/po/fr/Directories.po (hunk 2 above)
Comment (trebmuh) [reply _translator-files/po/fr/Directories.po]: fixed too as well (I think)

**Comment by ann0see** [comment]
Please merge if everything is ok here.

**Comment by ignotus666** [comment]
Ok, merging. I'm not letting the Italian translation of Directory in ("Cartella") however as I don't think it's the correct term. Changing it to fuzzy on Weblate and leaving a comment.

**Pull request merged**